scripts/release_notes/common.py

```diff
@@ -0,0 +1,238 @@
+from collections import namedtuple
+from os.path import expanduser
+import locale
+import subprocess
+import re
+import requests
+import os
+import json
+
+categories = [
```

Comment:
I've noticed that some categories/topics we use on TorchVision are missing. For example `code quality`, `tests`, `ci` etc. Shall we add them?

Reply:
more like we should remove `categories` and `topics` as they're not used at all in this script ^^

Reply:
I think we should maintain them because they allow us to keep track of Better Engineering efforts. We could update the script to remap them under an umbrella topic related to improving infra.

Reply:
> We could update the script to remap them under an umbrella topic related to improving infra.

What do you mean by that?

Reply:
FYI those categories were taken as is from PyTorch, and we haven't used them yet in our scripts. I can clean up this script further (it's mostly a dump of what I used for generating the release notes, not really cleaned up yet)